x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/application/connector/action/UpdateConnectorLastSeenAction.java

```diff
@@ -68,7 +63,7 @@ public ActionRequestValidationException validate() {
         public XContentBuilder toXContent(XContentBuilder builder, Params params) throws IOException {
             builder.startObject();
             {
-                builder.field(Connector.LAST_SEEN_FIELD.getPreferredName(), lastSeen);
+                builder.field(Connector.ID_FIELD.getPreferredName(), connectorId);
```

Comment:
Should we adapt the integration test using `is_after` similar to https://github.com/elastic/elasticsearch/blob/7f9ddd7f6ec517af9d4216f9a4c9b6b9038d0c9b/x-pack/plugin/ent-search/qa/rest/src/yamlRestTest/resources/rest-api-spec/test/entsearch/420_connector_sync_job_check_in.yml#L17-L43 to make sure that the new timestamp is after the old one?